lib/generators/avo/tailwindcss/install_generator.rb

```diff
@@ -27,12 +27,12 @@ def create_files
           end
 
 
-          unless (path = Rails.root.join("app", "assets", "stylesheets", "avo" ,"tailwind.css")).exist?
+          unless (path = Rails.root.join("app", "assets", "stylesheets", "avo" ,"avo.tailwind.css")).exist?
             say "Add default tailwind.css"
             copy_file template_path("avo.tailwind.css"), path
           end
 
-          script_name = "avo:tailwind:css"
```

Comment:
@adrianthedev since tailwind uses `tailwind:css` don't we want to follow that pattern and use `avo:tailwind:css` instead `avo:tailwindcss` ?

Reply:
nah!

1. we'll never have `tailwind:js` 😛 
2. and the name of the library is `tailwindcss`
3. we had that task name from the start
4. I don't see where they use it with semicolon

Reply:
> I don't see where they use it with semicolon

They don't. My bad, I had that wrong idea that they do use it with semicolon